components/mpas-ocean/bld/build-namelist

```diff
@@ -771,6 +771,8 @@ if ($OCN_ISMF eq 'coupled') {
 	add_default($nl, 'config_land_ice_flux_mode', 'val'=>"coupled");
 } elsif ($OCN_ISMF eq 'internal') {
 	add_default($nl, 'config_land_ice_flux_mode', 'val'=>"standalone");
+} elsif ($OCN_ISMF eq 'data') {
```

Comment:
I was looking through how we changed namelist settings through compsets with the data icebergs (grep around for `iceberg_mode`,  `MPASSI_ICEBERG_MODE`, `_MPASSI%.*DIB`), and we did it slightly differently, but this works equally well. If there is a need to change other namelist values based on `DISMF` we can explore that, but I think based on the discussion so far, you don't want things (e.g. GM) to change based solely on `DISMF`.

Reply:
No, that's right.  What I want is for WCYCL configurations to move to the ECwISC meshes and have DISMF and DIB, but to otherwise retain the same settings.  It would also be reasonable to run CRYO configs with DISMF, so I don't want DISMF to control any settings other than this one namelist option (and the stream to bring in the data it needs).